### PR TITLE
Mypy not auto on

### PR DESCRIPTION
**Pull request by gregwebs**

### CODE REVIEW

**Comment by gregwebs** [comment]
You might want to squash this

**Comment by gregwebs** [comment]
I tried to add MyPy to the README but I got crazy unfixable merge conflicts. Please add after merging.

**Comment by blueyed** [comment]
Please keep this in #439.

**Comment by gregwebs** [comment]
#439 is closed now, please re-open and merge this.

File: autoload/neomake/makers/ft/python.vim [hunk 1]
```diff
@@ -5,7 +5,10 @@ function! neomake#makers#ft#python#EnabledMakers()
         return s:python_makers
     endif
 
-    let makers = ['python', 'frosted']
+    let makers = ['python']
+    if executable('frosted')
+        call add(makers, 'frosted')
+    endif
```
Comment (blueyed) [inline autoload/neomake/makers/ft/python.vim]: That's unnecessary: the executable-check gets done on all makers.

Comment (gregwebs) [inline autoload/neomake/makers/ft/python.vim]: why do many of the other makers have this check here? It seems like all or none should have it.

Comment (gregwebs) [inline autoload/neomake/makers/ft/python.vim]: oh, I see, it looks like a technique to avoid having multiple linters.

**Comment by gregwebs** [comment]
ok, should be good to go now

**Comment by blueyed** [comment]
Thanks a lot!

**Pull request merged**